v2/sourcedb-to-spanner/src/test/java/com/google/cloud/teleport/v2/source/reader/io/jdbc/iowrapper/JdbcIoWrapperTest.java

```diff
@@ -248,20 +248,6 @@ public void testJdbcIoWrapperDifferentTables() throws RetriableSchemaDiscoveryEx
     String testCol = "ID";
     SourceColumnType testColType = new SourceColumnType("INTEGER", new Long[] {}, null);
     when(mockDialectAdapter.discoverTables(any(), any())).thenReturn(ImmutableList.of("testTable"));
-    when(mockDialectAdapter.discoverTableIndexes(any(), any(), any()))
```

Comment:
Is this deletion expected? It looks important and we are not removing this for the other tests?

Reply:
this was thrown up as an unused mock, so it has been removed.